### PR TITLE
chore: update akamai quickstart

### PR DESCRIPTION
**Pull request by jcountsNR**
small changes include renaming to correct name for datastream and slight updates to description for akamai.

### CODE REVIEW

File: quickstarts/datastream2-akamai/config.yml [hunk 1]
```diff
@@ -2,33 +2,31 @@ id: cb995039-39d8-44d9-97c3-58174a1337f2
...
-  - Cache hit retio 
-  - TLS overhead time ranges
-  - Overhead byte ranges
-  - HTTP protocol distribution
+    This integration utilizes the data captured and and stream via DataStream 2 to visualize and monitor your CDN performance and security logs in real time.
+    Included with the data
+    - HTTP status code distribution
+    - Error rate
+    - Log ingest
+    - Cache hit retio
```
Comment (RamanaReddy8801) [inline quickstarts/datastream2-akamai/config.yml:18]:
```suggestion
    - Cache hit ratio
```

**Review by RamanaReddy8801 — CHANGES_REQUESTED** [review]
Please look into the suggested changes.

**Comment by sarahkitten** [comment]
@RamanaReddy8801 can you approve this? committed your suggestion

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM!

**Pull request merged**